rfcs/text/0000-application-monitoring-for-ga.md

```diff
@@ -0,0 +1,146 @@
+### DCP PR:
+
+# System monitoring for General Availability
```

Comment:
I argue that this should not be labelled 'for General Availability'. It would be better to label this as simply 'System monitoring' and separately discuss component implementation plans.

Reply:
To clarify, the original intent of the document was to define what should be in place by GA.

If you feel strongly that these should be separate discussions, I will change the title and we can spin this up in the GA scoping discussions. If you have the sense that these things should be in place for GA. I'd assume just wrap up that discussion now.

Personally, I'd be apprehensive to operate a system publicly without at least an MVP of what is outlined here.

Reply:
My position, stemming from the GA doc, is that monitoring is not mandatory for ingest by GA since the process will still be wrangler mediated. Happy to have that discussion in tech arch but I think it would be better to have that conversation outside of this RFC. If you're happy with that title change then I would regard this as resolved. Otherwise, I would like to delay approval until we've talked this over with the rest of tech arch.

Reply:
@justincc - my earlier recommendation to @tburdett was to have a process for managing and tracking exceptions to mandatory requirements. For example, Technical Architecture could require a high-confidence schedule/plan for when a requirement would be addressed by a component.

Reply:
@justincc Brian's recommendation makes sense to me and I believe it addresses your core concern about adapting to where ingest is at. Are you OK with moving forward now?

Reply:
No I'm sorry. I would like to discuss any such exception-based plan with tech arch first.